narwhal/primary/src/primary.rs

```diff
@@ -723,27 +723,6 @@ impl PrimaryReceiverHandler {
             .sync_batches(header, network, /* max_age */ 0)
             .await?;
 
-        // Check that the time of the header is smaller than the current time. If not but the difference is
```

Comment:
@gdanezis added this in [PR#6056](https://github.com/MystenLabs/sui/pull/6056) and called out the condition of not signing headers in the past, what does removing this do to the correctness of the system? Should we just be adjusting the tolerance level based on the timestamp drift we are seeing?

Reply:
Good point where I looked at a different previous PR. In general I think a node should not slow down for an untrusted header. Will update the PR.